src/dropbox.atd

```diff
@@ -64,3 +64,5 @@ type metadata = { size: string;
                   ?client_mtime: date option;
                   root: root;
                   ~contents: metadata list }
+
+type revisions = metadata list
```

Comment:
This is needed for the functions generation but it does not mean it must be exported.